index.js

```diff
@@ -1,6 +1,6 @@
 let express = require("express")
 let Web3 = require('web3');
-let trader = require('./contracts/Trader.json')
+let traderABI = require('./contracts/Trader.json')
```

Comment:
if you want you can install @tracer-protocol/contracts to avoid this in the future

Reply:
I'll set that up on a new branch ty @dospore.